committee-rails.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.add_dependency 'committee', '>= 5.0.0'
-  spec.add_dependency 'activesupport', '>= 5.1'
-  spec.add_dependency 'actionpack', '>= 5.1'
-  spec.add_dependency 'railties', '>= 5.1'
+  spec.add_dependency 'committee', '>= 5.1.0'
+  spec.add_dependency 'activesupport', '>= 6.1'
+  spec.add_dependency 'actionpack', '>= 6.1'
+  spec.add_dependency 'railties', '>= 6.1'
```

Comment:
Since Rails 6.0 seems to work with Ruby 2.7, can you change the dependencies to `>= 6.0`?

Reply:
Sorry for the delay, changed to 6.0.